setup.cfg

```diff
@@ -92,6 +92,7 @@ docs =
     sphinx-autodoc-typehints
     sphinx_automodapi
     texext
+    numpy
```

Comment:
is this necessary? The whole package is installed to build the documentation (I'm pretty sure) so this should come along for the ride...

Reply:
maybe using `systempackages: true` also makes numpy available faster?

Reply:
Yes, it's because if it is not installed explicitly it runs into a numpy problem generating the search indices for the HTML documentation.
```
dumping object inventory... /home/docs/checkouts/readthedocs.org/user_builds/pykeen/envs/16/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
```

Reply:
Let's see if it passes now with `system_packages: true` (good catch on the underscore). If not, we can revert and then call this done

Reply:
Doesn't work. It still shows that error.

Reply:
If you scroll all the way down to the raw build file (https://readthedocs.org/api/v2/build/12074866.txt), it shows the warning

Reply:
But I checked the previous versions and there it occurred too. So given that the build is successful, we'll keep it this way now.